docs/src/content/docs/ops/backup-and-restore/backup-practices.adoc

```diff
@@ -0,0 +1,177 @@
+  - While XTDB uses immutable files, versioning remains an important safeguard and is a requirement of most available cloud backup tools.
+  - **S3**: link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html[S3 Versioning^]
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/blobs/versioning-overview[Blob Versioning^]
+  - **GCS**: link:https://cloud.google.com/storage/docs/object-versioning[Object Versioning^]
+* **Soft delete** — temporary protection against deletion
+  - Typically paired with versioning or retention policies.
+  - **S3**: _(Handled implicitly by versioning — deleting an object adds a delete marker, allowing recovery of previous versions)_
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/blobs/soft-delete-container-overview[Soft delete for containers^] 
+  - **GCS**: link:https://cloud.google.com/storage/docs/soft-delete[Soft delete^]
+* **Replication** — cross-region replication for resilience against regional failure
+  - **S3**: link:https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html[S3 Replication^]
+  - **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy[Zone Redundant / Geo-Redundant Storage^]
+  - **GCS**: link:https://cloud.google.com/storage/docs/locations#considerations[Dual-region / multi-region buckets^]
+
+These features act as a first line of defence — allowing recovery even before formal backups are consulted.
+
+=== Full Backups for Point-in-Time Recovery
+
+Full backups of the storage module are the recommended approach for recovering to a known good point in time, migrating XTDB environments, or in the case of a complete loss of object store.
+
+Because storage files are immutable once written, they are ideally suited to snapshot-based backup strategies.
+
+To perform a full backup:
+
+* Capture the full object store prefix or container used by XTDB, including all files written to date
+* Ensure the backup includes all files associated with the latest indexed block
+
+Cloud-native tools can help implement and automate storage backups — though support for point-in-time backups varies between providers:
+
+* **S3**: link:https://docs.aws.amazon.com/aws-backup/latest/devguide/whatisbackup.html[AWS Backup^] supports full point-in-time backups of S3 buckets, including versioned objects and lifecycle integration.
+* **Azure Blobs**: link:https://learn.microsoft.com/en-us/azure/backup/backup-overview[Azure Backup^] supports snapshot-based backups of blob storage, including integration with retention policies and soft delete.
+* **GCP**: Cloud Storage does not currently offer native snapshotting. Instead:
+  ** link:https://cloud.google.com/storage-transfer-service[Storage Transfer Service^] can be used to automate full or incremental copies across buckets or regions
+
+'''
+
+[#log-backup]
+== Backing Up the Log
```

Comment:
The lowercase t here amuses me, but I'm not pedantic enough to care either way :)